docker_run.sh

```diff
@@ -1 +1,17 @@
-docker run -it --rm --volume $(pwd)/results:/code/results --volume $(pwd)/map:/code/map grinder-framework -h
\ No newline at end of file
+#!/usr/bin/env sh
+
+pwd=$(pwd)
+
+if [ ! -e database.db ]; then
+	touch ${pwd}/database.db
+fi
+
+mkdir -p ./results/
+mkdir -p ./map/static/data/
+
+docker run -it --rm \
```

Comment:
Maybe it will be helpful, especially in the case of multiprocessing running (Nmap, scripts, etc.):
https://docs.docker.com/config/containers/resource_constraints/

We can discuss it.